source/nanoFramework.Hardware.Esp32/TouchPad/Enums.cs

```diff
@@ -0,0 +1,51 @@
+﻿
+namespace nanoFramework.Hardware.Esp32.TouchPad
```

Comment:
In C# enum values always show the enum name, like in `TouchFsmMode.TOUCH_FSM_MODE_SW`. This is to say that, unlike what's the convention for C, enumerations shouldn't start with the enum name and use PascalCasing. Check the guideline [here](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces#naming-enumerations).

The above should be something like: `TouchFsmMode.Software` 

Still on the enums:
- Make sure to add the comment for documentation using the expected format (above with /// and not in front of the value).
- It's OK to reference the IDF documentation, but that should come in the <remarks> section, not being _the_ explanation text by itself. It's OK if you copy/paste the description from there.